README.md

```diff
@@ -12,7 +12,7 @@ GraphQL endpoint providing a single endpoint to access data from Argo's many ser
 ### Quick Start
 
 - Set up environment: copy `.env.schema` to `.env` and update environment accordingly if needed. Values provided in the schema file can be used when running the server locally for development.
-- Install dependencies: `npm i`
+- Install dependencies: `npm ci` (depending on your Node version, it may require a '--legacy-peer-deps' flag)
```

Comment:
can we not suggest people do `npm i` anymore? makes life harder for maintainers.